include/clang/Parse/Parser.h

```diff
@@ -144,6 +144,25 @@ class Parser : public CodeCompletionHandler {
   mutable IdentifierInfo *Ident_final;
   mutable IdentifierInfo *Ident_override;
 
+  /// Checked C contextual keywords
+
+  /// These keywords are for bounds expressions.  They are contextual to avoid
+  /// collisions with existing identifiers in programs.  Some keywords like count
```

Comment:
Nit: I know this is being picky, but I'd recommend placing keywords like this in quotes or double-quotes when using them in comments; it just makes it easier for the reader to parse fragments like "and any are likely to collide".